silkworm/silkrpc/core/blocks.cpp

```diff
@@ -93,12 +93,12 @@ Task<std::pair<uint64_t, bool>> get_block_number(const std::string& block_id, co
     co_return std::make_pair(block_number, is_latest_block);
 }
 
-Task<uint64_t> get_block_number(const std::string& block_id, const rawdb::DatabaseReader& reader) {
+Task<BlockNum> get_block_number(const std::string& block_id, const rawdb::DatabaseReader& reader) {
```

Comment:
The same change here and in the rest of functions in the module should be applied also in `blocks.hpp`.

Reply:
ok